reboot_toolkit/toolkit_backend.py

```diff
@@ -233,15 +233,59 @@ def load_games_to_df_from_s3_paths(game_paths: list[str]) -> pd.DataFrame:
     for i, game_path in enumerate(game_paths):
 
         try:
-            current_game = wr.s3.read_csv(game_path, index_col=[0], use_threads=True).dropna(axis=1, how='all')
-
-            session_date_idx = [i for i, s in enumerate(game_path.split('/')) if s.isnumeric() and (len(s) == 8)][0]
-            current_game['session_date'] = pd.to_datetime(game_path.split('/')[session_date_idx])
-            print(current_game['session_date'].iloc[0])
-
-            session_num_idx = [i for i, s in enumerate(game_path.split('/')) if s.isnumeric() and (len(s) == 6)][0]
-            current_game['session_num'] = game_path.split('/')[session_num_idx]
-            print(current_game['session_num'].iloc[0])
+            if 'hitting-processed-series' in game_path:
+                swing_filenames = wr.s3.list_objects(game_path)
+                swing_dfs = []
+                for swing_filename in swing_filenames:
+                    swing_df = wr.s3.read_csv(swing_filename, index_col=[0], use_threads=True).dropna(axis=1, how='all')
+
+                    basepath = os.path.join(*swing_filename.split('/')[1:-2])
+                    movement_id = os.path.basename(swing_filename)[:-8]
+                    org_movement_id = movement_id.split('_')[-1]
+
+                    if not 'time' in swing_df:

```

Comment:
can you double check that this works as expected? I think you have to put swing_df.columns, otherwise it checks the index as opposed to the columns... but i'm not sure